.github/workflows/test.yml

```
@@ -0,0 +1,97 @@
+# ABOUTME: GitHub Actions workflow for running tests on every push and pull request
+# ABOUTME: Tests across multiple Python versions with comprehensive coverage reporting
+
+name: Tests
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)